Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR2</title>
+    <link rel="stylesheet" href="Styles.css">
```

Comment:
Be careful about capitalization. For projects actually displayed online, some servers care a great deal about capitalization of filenames:
```suggestion
    <link rel="stylesheet" href="styles.css">
```